### PR TITLE
UBER-274,-287,-288,-294: fixed tooltip, ActionsPopup, ListHeader, activity.

### PR DESCRIPTION
**Pull request by SasLord**
# Contribution checklist
Снимок экрана 2023-05-30 в 02 57 37 Снимок экрана 2023-05-30 в 00 34 35
Снимок экрана 2023-05-30 в 01 02 17 Снимок экрана 2023-05-30 в 02 28 42

## Brief description
UBER-274: fixed tooltip.
UBER-287: update activity.
UBER-288: fixed ActionsPopup.
UBER-294: added divider in ListHeader.

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues
UBER-274,-287,-288,-294

### CODE REVIEW

File: plugins/view-resources/src/components/list/ListHeader.svelte [hunk 1]
```diff
@@ -61,8 +61,9 @@
 
   let accentColor: ColorDefinition | undefined = undefined
 
+  $: showColors = (viewOptions as any).shouldShowColors
```
Comment (haiodo) [inline plugins/view-resources/src/components/list/ListHeader.svelte]:
```suggestion
  $: showColors = (viewOptions as any).shouldShowColors !== false
```
viewOptions?.shouldShowColors could be undefined, in this case it need to be true.

File: plugins/view-resources/src/components/list/ListHeader.svelte [hunk 2]
```diff
@@ -61,8 +61,9 @@
...
   $: headerBGColor =
-    level === 0 && (viewOptions as any).shouldShowColors !== false
+    level === 0 && showColors !== false
```
Comment (haiodo) [inline plugins/view-resources/src/components/list/ListHeader.svelte]:
```suggestion
    level === 0 && showColors
```

File: plugins/view-resources/src/components/ActionsPopup.svelte [hunk 3]
```diff
@@ -242,12 +244,20 @@
       on:close={async () => {
         activeAction = undefined
         await tick()
-        input?.focus()
+        input?.focused()
```
Comment (haiodo) [inline plugins/view-resources/src/components/ActionsPopup.svelte]: Why it is named focussed() ??

Comment (SasLord) [reply plugins/view-resources/src/components/ActionsPopup.svelte]: This is the name of the function in EditWithIcon, the other is occupied by the property

**Review by haiodo — APPROVED** [review]

**Pull request merged**